### ISSUE TITLE
Requesting modified output of report

### ISSUE THREAD

**Issue by minhtrung1997** [GunzIvan28/rMAP#6]
![image](https://user-images.githubusercontent.com/52932655/147314812-35beeb99-501d-45a6-8c17-32268a1085b6.png)

I've thought that this can cause problem generating report, I think it's should be erased of $HOME/ dir

**Comment by GunzIvan28** [comment]
@minhtrung1997, thanks for using the pipeline. The reason why a central report directory is used is because R and it's libraries communicate better in that way. All results are collated and aggregated from there. 

In case you still have a better reason why you feel it should be scrapped off, kindly provide more insight.


Ivan

**Comment by minhtrung1997** [comment]
Yeah I have fix the bug and change some to fit our team requirements
It would be nice if we can discuss more for the development of this pipeline.

Best Regards

Minh Trung

**Comment by GunzIvan28** [comment]
Hey @minhtrung1997,

Thanks for the feedback. We can have a zoom and chat more. Send me am email via:

_ivangunz23@gmail.com_

We can set up a meeting at your earliest convenience.


Cheers,
Ivan

**Comment by minhtrung1997** [comment]
Thank you. I appreciate it. For now, I'm a bit busy so I can't contact you in detail. But when I'm free in the next month, I also want to contact you through the mail and also discord.
Best regard

**Comment by GunzIvan28** [comment]
Alright, will wait to hear from you soon.

**Comment by minhtrung1997** [comment]
Anyway, why don't you create a next flow version of this pipeline?

**Comment by GunzIvan28** [comment]
@minhtrung1997 , the nextflow version is already under development and currently undergoing testing. It will be deployed as soon as a manuscript for it is ready

**Comment by minhtrung1997** [comment]
I have a break now with my company. We can set up a short casual online meeting from today till Saturday.😊😊😊

**Comment by GunzIvan28** [comment]
Great, reach out via: ivangunz23@gmail.com and we set up a meet.
